src/customerBookings.js

```diff
@@ -0,0 +1,18 @@
+function getCustomerBookings(customerId, bookings) {
+  const customerBookings = bookings.filter(booking => customerId === booking.userID);
+  return customerBookings;
+}
+
+function calculateTotalCost(bookings, rooms) {
+  const totalCost = bookings.reduce((total, booking) => {
```

Comment:
Not necessary, but to save a line of code you could return bookings.reduce on line 7 and remove line 12.